pkg/cli/admin/upgrade/recommend/examples/4.14.1-all-recommended.output

```diff
@@ -1,5 +1,7 @@
 info: An update is in progress.  You may wish to let this update complete before requesting a new update.
   Working towards 4.14.1: 139 of 859 done (16% complete), waiting on kube-scheduler
+The following conditions found no cause for concern in updating this cluster to later releases: recommended/Alert (NoTestData)
```

Comment:
Nit: We could use a blank line to separate from whatever `Progressing=True` says?

Reply:
works for me, done with 1c84ceaf6 -> 426959c86.